homeassistant/components/baf/const.py

```diff
@@ -0,0 +1,28 @@
+"""Constants for the Big Ass Fans integration."""
+from enum import IntEnum
+
+DOMAIN = "baf"
+
+# Most properties are pushed, only the
+# query every 5 minutes so we keep the RPM
+# sensors up to date
+QUERY_INTERVAL = 300
+
+RUN_TIMEOUT = 20
+
+PRESET_MODE_AUTO = "Auto"
+
+SPEED_COUNT = 7
+SPEED_RANGE = (1, SPEED_COUNT)
+
+ONE_MIN_SECS = 60
+ONE_DAY_SECS = 86400
+HALF_DAY_SECS = 43200
+
+
+class OffOnAuto(IntEnum):
+    """Tri-state mode enum that matches the protocol buffer."""
+
+    OFF = 0
+    ON = 1
+    AUTO = 2
```

Comment:
This needs to come out since the upstream lib can provide it (currently had a bug in the upstream lib)

Reply:
bumped lib.  This is resolved now